load.go

```diff
@@ -471,22 +490,22 @@ func (c *TestSpec) processBuildArgs(lex *shell.Lex, args map[string]string, name
 	for i, step := range c.Steps {
 		stdout := step.Stdout
 		if err := stdout.processBuildArgs(lex, args); err != nil {
-			return err
+			errL.Append(err)
 		}
 		step.Stdout = stdout
 
 		stderr := step.Stderr
 		if err := stderr.processBuildArgs(lex, args); err != nil {
-			return err
+			errL.Append(err)
 		}
-		step.Stderr = stderr
 
+		step.Stderr = stderr
 		c.Steps[i] = step
 	}
 
 	for name, f := range c.Files {
 		if err := f.processBuildArgs(lex, args); err != nil {
-			return errors.Wrap(err, name)
+			errL.Append(errors.Wrap(err, name))
```

Comment:
The error is never returned in this case now.

Reply:
Thank you for catching this!